src/core_plugins/kibana/public/dashboard/dashboard_app.js

```diff
@@ -37,6 +38,10 @@ const app = uiModules.get('app/dashboard', [
   'kibana/typeahead',
 ]);
 
+app.directive('dashboardListing', function (reactDirective) {
```

Comment:
What do you think about moving this to index.js? This file isn't related to the listing page, it only works because we always import this file in index.js, but it feels accidental.

We do then end up with grabbing the app twice.  I'm actually not sure how that works... if you call `uiModules.get('app/dashboard', [stuff])` two different times but stuff differs.  Do you only need to pass it stuff that is used on that page?  Or should it always get passed everything? If the former, I think in index.js you could load it up with just react.  An angular question that I'm not sure of the answer too... I'd have to dig into it a bit. Or ask an angular expert on our team (@spalger ?)